### ISSUE TITLE
Jruby node variants?

### ISSUE THREAD

**Issue by deivid-rodriguez** [circleci/circleci-images#266]
Hello CircleCI team!

I noticed that `-node` and `-node-browsers` variants in `jruby` are not available like they are in MRI. Would you be willing to add those? This can be useful for gems considering moving to CircleCI that test on different implementations (I'm currently experimenting with migrating [activeadmin](https://circleci.com/workflow-run/3f10d509-8309-4023-9cc6-7228e1b6706c).

Thanks a lot! :+1:

**Comment by felicianotech** [comment]
Correct, only some of our images have been equipped with a Node variant.

We'll consider this and get back to you shortly on this.

**Comment by deivid-rodriguez** [comment]
Thanks os much for getting back to me and for considering :+1:.

**Comment by ryanwohara** [comment]
PR is here
https://github.com/circleci/circleci-images/pull/267/files

When it's merged it will be in our staging account initially.

**Comment by deivid-rodriguez** [comment]
@ryanwohara That means the built image will not yet be publicly available or that I would need to pull it from a different location?

**Comment by deivid-rodriguez** [comment]
I found the answer to my question, I'm pulling the image from staging for now, thanks! Shall I close this ticket or wait until it's available on the "official" repo?

**Comment by iynere** [comment]
@deivid-rodriguez awesome, glad to hear it! we'll merge it into master shortly.

took a look at the link re: certain jruby versions not being available—if the official jruby maintainers start actively building those versions, we will at some point afterwards pull that from their manifests & also build those versions

**Comment by deivid-rodriguez** [comment]
@iynere They've been already [added](https://github.com/docker-library/official-images/pull/4971)! :smile:

**Comment by deivid-rodriguez** [comment]
Ok, now I have this issue where https://github.com/docker-library/official-images/pull/4971 has been picked up by circle-dockerfiles [master branch](https://github.com/CircleCI-Public/circleci-dockerfiles/tree/master/jruby/images), but not by the [staging branch](https://github.com/CircleCI-Public/circleci-dockerfiles/tree/staging/jruby/images). Is that something that should be synced manually or will eventually happen automatically? Thanks!

**Comment by deivid-rodriguez** [comment]
That was indeed the issue, I see that the changes were picked up now that you've pushed to that branch and everything is [passing now](https://circleci.com/workflow-run/9802cf6a-3815-4061-8a7e-527c95fdf253). Thanks so much!
